.github/workflows/package-command.yaml

```diff
@@ -11,18 +11,26 @@ env:
   ADO_TEAMPROJECT: "${{ vars.ADO_TEAMPROJECT }}"
   ADO_PARENT_LINK: "${{ vars.ADO_PARENT_LINK }}"
   GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+  PR_NUMBER: ${{ github.event.client_payload.pull_request.number && github.event.client_payload.pull_request.number || github.event.client_payload.pullRequestNumber }}
+  RUN_ID: ${{ github.run_id }}
 
 on:
   repository_dispatch:
     types: [package-command]
 jobs:
   solutionNameDetails:
+    permissions:
```

Comment:
@v-amolpatil, can we add these permissions globally above jobs so that it should not require to add at each step.